.github/workflows/cygwin.yml

```
@@ -243,7 +243,7 @@ jobs:
         shell: bash.exe -eo pipefail -o igncr "{0}"
         id: cygwin-run-pytest
         run: |
-          xvfb-run pytest -rfEsXR -n auto \
+          xvfb-run python -m pytest -rfEsXR -n auto \
```

```suggestion
          xvfb-run pytest -rfEsXR -n auto \
```

See changes in https://github.com/matplotlib/matplotlib/pull/27552/files for details.  This is subtle interaction between pytest's import magic / test discovery + meson's build magic.

---

Well that explains the import errors once it found pytest.